src/qt/balancesdialog.h

```diff
@@ -58,24 +46,22 @@ class BalancesDialog : public QDialog
     QMenu *contextMenuSummary;
 
     GUIUtil::TableViewLastColumnResizingFixer *borrowedColumnResizingFixer;
-    virtual void resizeEvent(QResizeEvent* event);
+    virtual void resizeEvent(QResizeEvent *event);
```

Comment:
C noob question incoming - I thought ```(QResizeEvent* event)``` and ```(QResizeEvent *event)``` were no different - what does this change do?